pkg/pravega/pravega_segmentstore.go

```diff
@@ -34,6 +35,21 @@ const (
 )
 
 func deploySegmentStore(pravegaCluster *api.PravegaCluster) (err error) {
+	err = sdk.Create(makeSegmentStoreHeadlessService(pravegaCluster))
+	if err != nil && !errors.IsAlreadyExists(err) {
+		return err
+	}
+
+	if pravegaCluster.Spec.ExternalAccess {
+		services := makeSegmentStoreLoadBalancerServices(pravegaCluster)
```

Comment:
This logic will probably require revision once we support scaling of the segment stores.  Since you'd need to create/delete service resources as needed.